### ISSUE TITLE
Swiping with multiple fingers triggers scrolling

### ISSUE THREAD

**Issue by waissbluth** [alvarotrigo/fullPage.js#2540]
### Description
On mobile devices, the scrolling to another slide is triggered even when touching with two or more fingers. This is problematic, for example, when having an embedded Google Map with [Cooperative Gesture Handling](https://developers.google.com/maps/documentation/javascript/examples/interaction-cooperative)

I think that any kind of multi-finger gesture shouldn't be triggering any slide/section events.

Is there a currently way to prevent this, either entirely or via an option?

### Link to isolated reproduction with no external CSS / JS
You can try it in http://alvarotrigo.com/fullPage/

### Steps to reproduce it
1. Load a fullPage site with touch scrolling enabled
2. Swipe down with two or more fingers

### Versions 
Chrome for Android, Chrome for iOS, Safari for iOS

Thanks

**Comment by alvarotrigo** [comment]
> Is there a currently way to prevent this, either entirely or via an option?

Not at the moment I'm afraid. I'll mark it as a possible enhancement. 

I guess you get exactly the same result with the [Drag And Move example](http://alvarotrigo.com/fullPage/extensions/drag-and-move.html) too?

**Comment by waissbluth** [comment]
Yes, the Drag And Move example is also triggered by multiple finger gestures.

I guess the `touches` [attribute of the touch event](https://developer.mozilla.org/en-US/docs/Web/API/TouchEvent/touches) is a place to start considering this, but I don't know about browser support for this.

**Comment by waissbluth** [comment]
Actually, could this just be implemented as a condition in https://github.com/alvarotrigo/fullPage.js/blob/master/jquery.fullPage.js#L1100 ?
